src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -70,6 +65,12 @@
     <IncludeDefaultReferences Condition="'$(MSBuildProjectExtension)' == '.vbproj'">true</IncludeDefaultReferences>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true'">
+    <IncludeTestReferences Condition="'$(IncludeTestReferences)' == ''">true</IncludeTestReferences>
+    <IncludePerformanceTestReferences Condition="'$(IncludePerformanceTestReferences)' == ''">true</IncludePerformanceTestReferences>
```

Comment:
This seems wrong.  ALL test projects are going to get perf test references?

Reply:
This is the current behavior but no, I'm right now this moment changing this.